li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/largemessage/LargeMessageSegment.java

```diff
@@ -46,6 +49,14 @@ public LargeMessageSegment(UUID messageId,
     this.messageSizeInBytes = messageSizeInBytes;
     this.payload = payload;
   }
+  public LargeMessageSegment(LargeMessageHeaderValue headerValue,
```

Comment:
Can you use the existing constructor, instead of passing in a `LargeMessageHeaderValue` ?

Reply:
Updated with using existing constructor